### DISCUSSION TITLE
Marcha Imperial Canina

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#456] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/marcha-imperial-canina/

**Comment by one-eyed-willy** [comment]
Uma parte de mim realmente duvida de que tenha sido apenas puro adestramento, mas se realmente foi, caaaara...

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/videos/marcha-imperial-canina/#comment_1087628251>):

> Uma parte de mim realmente duvida de que tenha sido apenas puro adestramento, mas se realmente foi, caaaara...

Eu até curto ver bastidores desse tipo de coisa, mas evito pra não acabar me desapontando
